### DISCUSSION TITLE
Generate an intermediary report

### DISCUSSION THREAD

**Discussion by bistace** [ERGA-consortium/EARs#35] [Ideas]
Hello 👋,

I would like to generate the report on non-curated assemblies to see if there is a problem even before scaffolding, as Hi-C data often come a bit after long-reads data. 

I have seen that the `make_ear.py` script is looking for the `Curated` tag in the yaml file and was wondering if it would be possible to implement the generation of the report without it.

**Comment by diegomics** [comment]
Hi @bistace
During the initial designing and testing phase of the EARs development, we were producing pre-curation reports with information of the assembly steps divided in _contigging_ and _scaffolding_, and also showing other information (see [this](https://github.com/ERGA-consortium/EARs/blob/main/Assembly_Reports/Rupicapra_rupicapra/mRupRup1/mRupRup1_preEAR.pdf)). All that development is now in the preCuration_test branch of this repo.
I would like to revive that as a resource for helping in situations like yours.
I will update the code to display better information and some things that were missing and create a folder inside the main branch with all the explanations.
It will take me a few days.

**Reply by bistace** [reply]
Very nice, thank you!
